raft.c

```diff
@@ -671,7 +672,7 @@ raft_cbs_t redis_raft_callbacks = {
     .persist_vote = raftPersistVote,
     .persist_term = raftPersistTerm,
     .log = raftLog,
-    .log_get_node_id = raftLogGetNodeId,
+    .get_node_id = raftLogGetNodeId,
```

Comment:
this is actually concerning.  did I somehow mess up libraft and change the name or in the midst of rebases, a change got reverted by me that we didn't catch?